docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
```

Comment:
since we now know how to switch branches, do we still need Makefile?

Reply:
I'd say let's keep this makefile so that we can preview the build locally... It's also much easier to debug the doc build with the makefile.